src/platform/qt/MemorySearch.cpp

```diff
@@ -125,9 +126,16 @@ bool MemorySearch::createParams(mCoreMemorySearchParams* params) {
 	}
 	if (m_ui.typeStr->isChecked()) {
 		params->type = mCORE_MEMORY_SEARCH_STRING;
-		m_string = m_ui.value->text().toLocal8Bit();
+		QString enc = m_ui.encoding->currentText();
```

Comment:
The state of this not be stored in the UI itself; please store this as a variable on the search itself.